src/components/PlaylistTable.jsx

```diff
@@ -0,0 +1,139 @@
+import React from "react"
+import $ from "jquery" // TODO: Remove jQuery dependency
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome"
+
+import PlaylistRow from "./PlaylistRow"
+import Paginator from "./Paginator"
+import PlaylistsExporter from "./PlaylistsExporter"
+import { apiCall } from "helpers"
+
+class PlaylistTable extends React.Component {
+  state = {
+    playlists: [],
+    playlistCount: 0,
+    likedSongsLimit: 0,
+    likedSongsCount: 0,
+    nextURL: null,
+    prevURL: null
+  }
```

Comment:
should be defined in your constructor:
https://reactjs.org/docs/react-component.html#constructor

Reply:
I read that if the constructor isn't required for anything else, this is cleaner e.g.:

https://dev.to/ascorbic/class-fields-are-coming-heres-what-that-means-for-react--3a87